FlowCrypt/Functionality/Services/Remote Pub Key Services/PubLookup.swift

```diff
@@ -14,6 +14,16 @@ class PubLookup: PubLookupType {
     private let wkd: WkdApiType
     private let attesterApi: AttesterApiType
 
+    private enum PubLookupSource {
+        case attester
+        case wkd
+    }
+
+    private struct Result {
```

Comment:
Swift already has built-in `Result` type (https://developer.apple.com/documentation/swift/result), it'll be better to rename this one to something like `LookupResult`